hydroshare/local_settings.template

```diff
@@ -61,15 +171,37 @@ HS_IRODS_LOCAL_ZONE_DEF_RES='hydroshareLocalResc'
 HS_WWW_IRODS_ZONE='hydroshareZone'
 HS_USER_IRODS_ZONE='hydroshareuserZone'
 
-# Override these values for Prod or anywhere with NGINX and support for large files
+# Email configuration
+EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+#EMAIL_HOST_USER = ''
+#EMAIL_HOST_PASSWORD = ''
+#EMAIL_HOST = ''
+#EMAIL_PORT = ''
+#EMAIL_USE_TLS = True
+#DEFAULT_FROM_EMAIL = ''
+#DEFAULT_SUPPORT_EMAIL=''
+
+HYDROSHARE_SHARED_TEMP = '/shared_tmp'
+
+#used by the mailchimp subscription job in hs_core/tasks.py
+MAILCHIMP_ACTIVE_SUBSCRIBERS = "e210a70864"
+MAILCHIMP_SUBSCRIBERS = "f0c27254e3"
+
+# sendfile support for large files
+# These must match settings in nginx
 SENDFILE_ON = False
 IRODS_USER_URI = "/irods-user"
 IRODS_DATA_URI = "/irods-data"
 LOCAL_CACHE_URI = "/local-cache"
 
-# Needed for deployments to non-dev environments
+RECAPTCHA_SITE_KEY="6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI"
+RECAPTCHA_SECRET_KEY="6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe"
+RECAPTCHA_VERIFY_URL='https://www.google.com/recaptcha/api/siteverify'
+
 with open(os.path.dirname(os.path.abspath(__file__))  + "/../config/hydroshare-config.yaml", 'r') as stream:
```

Comment:
You should be able to remove this block.  It's not used yet and was only added to expose some configuration for @phuongdm.  It's not needed in a local dev environment.

Reply:
Fixed in 1e92768ca